libs/blocks/susi-light/susi-light.js

```diff
@@ -0,0 +1,138 @@
+/* eslint-disable consistent-return */
+import { loadScript, createTag, getConfig, loadIms, isSignedOut } from '../../utils/utils.js';
+
+const loadSusiLight = async (env) => {
+  const lib = `https://auth-light.identity${env.name === 'prod' ? '' : '-stage'}.adobe.com/sentry/wrapper.js`;
+  await loadScript(lib);
+};
+
+const redirectIfLoggedIn = async (destURL) => {
+  const redirect = () => window.location.replace(destURL);
+  if (!isSignedOut()) return redirect();
+  try {
+    if (!window.adobeIMS) await loadIms();
+    if (window.adobeIMS?.isSignedInUser()) return redirect();
+  } catch (e) {
+    window.lana?.log(`Product Login pages: Unable to load IMS: ${e}`);
+  }
+};
+
+const onRedirect = (e) => window.location.replace(e.detail);
+
+const onError = (e) => window.lana?.log('Product Login pages: SUSI Light Error: ', e);
+
+export class SusiLight {
+  constructor(el) {
+    this.el = el;
+    this.children = el.querySelectorAll(':scope > div');
+    this.isDesktop = window.matchMedia('(min-width: 900px)');
+  }
+
+  getRedirectURL = (env) => this.children[5]?.textContent?.trim() || `https://www.${env.name === 'prod' ? '' : 'stage.'}adobe.com/home`;
+
+  init = async () => {
+    const { env } = getConfig();
+    redirectIfLoggedIn(this.getRedirectURL(env));
+    await loadSusiLight(env);
+    this.isDesktop.addEventListener('change', this.handleViewportChange);
+    this.setBackground();
+    this.el.innerHTML = '';
+
+    const loginContainer = this.createLoginContainer();
+    const productInfo = this.createProductInfoElement();
+
+    this.el.appendChild(loginContainer);
+    this.el.appendChild(productInfo);
+  };
+
+  handleViewportChange = () => this.setBackground();
+
+  createAuthParams = () => {
+    const { imsClientId, locale, imsScope, env } = getConfig();
+    return {
+      client_id: imsClientId,
+      scope: imsScope || window.adobeid.scope || 'AdobeID,openid,gnav',
+      response_type: 'token',
+      redirect_uri: this.getRedirectURL(env),
+      locale: locale?.ietf || 'en-US',
+    };
+  };
+
+  createSusiElement = () => {
+    const sentry = createTag('susi-sentry-light');
+    sentry.stage = true;
+    sentry.variant = 'standard';
+    sentry.authParams = this.createAuthParams();
+    sentry.config = {};
```

Comment:
Since createTag support attributes in second param, maybe pass it like this
```suggestion
    const sentry = createTag('susi-sentry-light', {
    stage: true,
    variant: 'standard',
    authParams: this.createAuthParams(),
    config: {},
  });
```